src/main/python/analyzers/emotion_analyzer.py

```diff
@@ -25,4 +25,7 @@ def getEmotionAnalysis(dataframe):
     :param dataframe: player dataframe with tweet column
     :return: A pandas dataframe with the emotion affects appended to the original dataframe
     """
-    return pd.concat([dataframe, pd.DataFrame([__getEmotions(row.Tweet) for i, row in dataframe.iterrows()])], axis=1)
+    return pd.concat([dataframe,
```

Comment:
it's not about looking better, i had to assign the index of the newly concatenated dataframe the same index values of the original dataframe in order to join both dataframe. otherwise, the method returns a dataframe with null for the columns 'date', 'tweet', 'pointsplusminus', 'accplusminus'. the new code should be kept.

Reply:
Oh, I see! I didn't notice you changed the index. Thanks for clarifying!